server-spi-private/src/main/java/org/keycloak/migration/MigrationModelManager.java

```diff
@@ -116,7 +118,7 @@ public static void migrateImport(KeycloakSession session, RealmModel realm, Real
             stored = new ModelVersion(rep.getKeycloakVersion());
```

Comment:
This comment below is from Bill.  Looks like it's fixed, though I did have to add a null check to make it work.

Do any of you know what this is about?  Does it look correct?